org.eclipse.xtext.common.types/build.gradle

```diff
@@ -9,3 +9,5 @@ dependencies {
 	optional "org.eclipse.emf:org.eclipse.emf.codegen:$versions.emfEcore"
 	optional "org.eclipse.emf:org.eclipse.emf.codegen.ecore:$versions.emfEcore"
 }
```

Comment:
build.properties?

Reply:
Good catch. Forgotten since the projects are build with Gradle and build.properties is only relevant for Eclipse builds. Will update ASAP.

Reply:
updated for this plugin